src/app/pokemon-list/pokemon-list.component.html

```diff
@@ -6,18 +6,21 @@
   ></app-pagination>
 </div>
 <div class="centerLoader">
-  <mat-spinner *ngIf="loading; else content"></mat-spinner>
+  @if (loading) {
+    <mat-spinner></mat-spinner>
+  } @else {
+    @if (pokemons.length > 0) {
+      <div class="container">
+        @for (pokemon of pokemons; track pokemon) {
```

Comment:
Why sometimes u track pokemons by id and sometimes not?

Reply:
yeah, my bad!
368d7e0